WordPress/src/main/res/values/strings.xml

```diff
@@ -2083,22 +2083,22 @@
 
     <!-- activity titles -->
     <string name="reader_title_applog">Application log</string>
-    <string name="reader_title_blog_preview">Reader Site</string>
-    <string name="reader_title_tag_preview">Topic</string>
+    <string name="reader_activity_title_blog_preview">Reader Blog</string>
+    <string name="reader_activity_title_tag_preview">Tag</string>
     <string name="reader_title_post_detail">Reader Post</string>
     <string name="reader_title_related_post_detail">Related Post</string>
-    <string name="reader_title_subs">Manage Topics &amp; Sites</string>
+    <string name="reader_activity_title_manage_tags_blogs">Manage Tags &amp; Blogs</string>
     <string name="reader_title_photo_viewer">%1$d of %2$d</string>
     <string name="reader_title_comments" translatable="false">@string/comments</string>
-    <string name="reader_title_interests">Follow topics</string>
+    <string name="reader_title_tags">Follow tags</string>
 
     <!-- view pager titles -->
-    <string name="reader_page_followed_tags">Followed topics</string>
-    <string name="reader_page_followed_blogs">Followed sites</string>
+    <string name="reader_page_followed_tags_title">Followed tags</string>
+    <string name="reader_page_followed_blogs_title">Followed blogs</string>
```

Comment:
❓  Is it ok to keep "follow"/"followed" in this situation or should it be replaced with "Subscribe"/"Subscribed"?

Reply:
Good catch, should be replaced